website/docs/cdktf/concepts/assets.mdx

```diff
@@ -38,16 +38,16 @@ class MyStack extends TerraformStack {
       region: "us-west-2",
     });
 
-    const bucket = new S3Bucket(this, "bucket", {
+    const bucket = new s3.S3Bucket(this, "bucket", {
       bucket: "demo",
     });
 
     const asset = new TerraformAsset(this, "lambda-asset", {
-      path: path.resolve(__dirname, "../lambda"),
+      path: os.path.join(os.path.dirname(__file__), "..", "lambda"),
```

Comment:
```suggestion
      path: path.join(path.dirname(__file__), "..", "lambda"),
```
(or change the import above)

Reply:
I think we need to change the import instead – or change the import in the example